### PR TITLE
slasher grpc server

### PR DESCRIPTION
**Pull request by shayzluf**
[Part of] #3133
---

# Description
Make slasher run as a separate server

### CODE REVIEW

File: slasher/rpc/server.go [hunk 1]
```diff
@@ -55,12 +56,10 @@ func (ss *Server) IsSlashableBlock(ctx context.Context, psr *ethpb.ProposerSlash
 
 // SlashableProposals is a subscription to receive all slashable proposer slashing events found by the watchtower.
 func (ss *Server) SlashableProposals(req *types.Empty, server ethpb.Slasher_SlashableProposalsServer) error {
-	//TODO(3133): implement stream provider for newly discovered listening to slashable proposals.
```
Comment (nisdas) [inline slasher/rpc/server.go]: why remove this ? it is still unimplemented

File: slasher/flags/flags.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	if home := os.Getenv("HOME"); home != "" {
+		return home
+	}
+	if usr, err := user.Current(); err == nil {
+		return usr.HomeDir
+	}
+	return ""
+}
+
+func defaultValidatorDir() string {
```
Comment (nisdas) [inline slasher/flags/flags.go]: validator dir ?

File: slasher/rpc/server.go (hunk 1 above)
Comment (shayzluf) [reply slasher/rpc/server.go]: i guess bad merge

Comment (shayzluf) [reply slasher/rpc/server.go]: getting it back

File: slasher/flags/flags.go (hunk 2 above)
Comment (shayzluf) [reply slasher/flags/flags.go]: removed unused code

Comment (shayzluf) [reply slasher/flags/flags.go]: thanks

File: slasher/main.go [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+	cmd.VerbosityFlag,
+	cmd.DataDirFlag,
+	cmd.EnableTracingFlag,
+	cmd.TracingProcessNameFlag,
+	cmd.TracingEndpointFlag,
+	cmd.TraceSampleFractionFlag,
+	cmd.BootstrapNode,
+	cmd.MonitoringPortFlag,
+	cmd.LogFileName,
+	cmd.EnableUPnPFlag,
```
Comment (nisdas) [inline slasher/main.go]: why do you need this flag ?

Comment (shayzluf) [reply slasher/main.go]: removed it.
thanks

**Pull request merged**